sdk/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## [Unreleased]
 
+## 3.0.0-alpha.20 [2023-01-09]
+
+- Remove the Cbor trait and its uses
+- Refactor: `send` uses `Option<IpldBlock>` for return value
```

Comment:
I excluded #1348 and #1383 from the changelog since the only changes to the SDK in those PRs are comments / docs.